visidata/loaders/archive.py

```
@@ -10,11 +10,13 @@
 
 @VisiData.api
 def guess_zip(vd, p):
+    if p.is_dir(): return
```

This shouldn't be required for guessers; we let it raise the exception and it should be silently ignored.

---

Okay. My aim was to clean up the `--debug` results. But very few people will ever see those. And I am seeing that visidata style is to take full advantage of ignoring exceptions in such situations, and keep the code lean.